tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -0,0 +1,199 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import requests
+from typing import TYPE_CHECKING
+
+from .config import PROXY_URL
+
+if TYPE_CHECKING:
+    from typing import Any, Dict
+
+
```

Comment:
I think we can have a general add_sanitizer()  which including all add_XXX_sanitizer(), it would be easier to call in test proxy migration.

Reply:
I would like a single method, but it gets complicated to detail how the various kwargs should be used, depending on what kind of sanitizer you want. Splitting the method into one for each sanitizer makes it a lot easier to document any parameters you can provide and how they should be used, which is why I went for this.

I guess we could have a general method call the other methods based on the sanitizer type provided, yeah... I just wonder if that extra call layer would hide the parameter documentation and make things confusing. What do you think?